vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -14,6 +14,10 @@ def index
     def new
     end
 
+    def backend?
+      true
```

Comment:
Every action the `ProjectsController` handles currently should be considered backend, giving us the modal order detail popup.